doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -644,6 +657,10 @@ OSSL_CMP_CTX_get_status(), and
 OSSL_CMP_CTX_get_failInfoCode()
 return the intended value as described above or -1 on error.
 
+OSSL_CMP_certConf_cb() returns B<fail_info> if it is not equal to B<0>,
```

Comment:
Should be `I<fail_info>`. There are other cases in the file where B should be replaced with I when referencing function arguments.

Reply:
I've replaced in that file all wrong usages of `B<...>` (typically by `I<...>` if these were argument references).